contrib/api-stress-test/clean.yml

```diff
@@ -1,11 +1,29 @@
 ---
+- hosts: all
+  gather_facts: False
+  tasks:
+    - name: setup
+      setup:
+      delegate_to: localhost
+
+    - name: set ansible control host IP fact
+      set_fact:
+        local_home_path: "{{ hostvars[inventory_hostname]['ansible_env']['HOME'] }}"
+      delegate_to: 127.0.0.1
```

Comment:
change to `localhost`?

Reply:
https://github.com/microsoft/pai/blob/master/contrib/api-stress-test/roles/virtual-kubelet/install/defaults/main.yml#L4 This role is not a localhost specific role. And in other hosts, this facts will be used.

Reply:
Done